headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,94 @@
+#ifndef HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+#define HEADERS_LIVE_ACTOR_GROUND_MODE_H_
+
+struct LiveActor {
+    uint16_t id;     /* The id of the Actor in the actor list */
+    uint16_t kind;   /* The specie of the Actor */
+    bool is_enabled; /* true when the actor is loaded, false otherwise (should be checked if unsure)
+                      */
+    undefined field3_0x5;
```

Comment:
Can we not number these? Just use the offset for unknown fields, i.e. `field_0x5`